src/main/java/se/iths/crimedatabase/security/SecurityConfig.java

```diff
@@ -11,22 +11,26 @@
 @Configuration
 @EnableWebSecurity
 public class SecurityConfig extends WebSecurityConfigurerAdapter {
-
-    private final String[] urls = {"/addresses", "/categories", "/crimes", "/criminals", "/users", "/victims", "/publish"};
-
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http
-                .csrf()
-                .ignoringAntMatchers(urls)
+                .csrf().disable()
+                .authorizeRequests()
+                .antMatchers("/images/**").permitAll()
+                .antMatchers("/showCrimes").hasRole("ADMIN")
+                .antMatchers("/showUsers").hasRole("ADMIN")
+                .antMatchers("/showCategories").hasRole("ADMIN")
+                .antMatchers("/showVictims").hasRole("ADMIN")
+                .antMatchers("/showCriminals").hasRole("ADMIN")
+                .antMatchers("/showAddresses").hasRole("ADMIN")
```

Comment:
What are theses "show..." parts, and why are they reserved for admins?

Reply:
I looked at some other pull requests and saw the /show... parts. Since they are not implemented in main right now this should be a draft or at least we need to know that we should not merge this in to main untill the other parts are done.